logzio-sender/src/test/java/io/logz/sender/LogzioLongRunningTests.java

```diff
@@ -57,18 +59,60 @@ public void testDeadLock() throws Exception {
         String loggerName = "deadlockLogger";
         int drainTimeout = 1;
         Integer gcInterval = 1;
-        LogzioSender logzioSender = getTestLogzioSender(token, type, drainTimeout, gcInterval, mockListener.getPort());
+        final int msgCount = 100000000;
+        File tempDir = TestEnvironment.createTempDirectory();
+        tempDir.deleteOnExit();
+        SenderStatusReporter reporter = new LogzioTestStatusReporter(logger);
+        HttpsRequestConfiguration conf = getHttpsRequestConfiguration(token, type, mockListener.getPort());
+        LogzioSender logzioSender =
+                LogzioSender
+                        .builder()
+                        .setDrainTimeout(drainTimeout)
+                        .setReporter(reporter)
+                        .setHttpsRequestConfiguration(conf)
+                        .WithDiskMemoryQueue()
+                        .setGcPersistedQueueFilesIntervalSeconds(gcInterval)
+                        .setBufferDir(tempDir)
+                        .EndDiskQueue()
+                        .setTasksExecutor(tasks)
+                        .build();
+        logzioSender.start();
+        sendLogs(loggerName, logzioSender, msgCount);
+    }
 
+    @Test
+    public void testInMemoryLongRun() throws Exception {
+        String token = "aBcDeFgHiJkLmNoPqRsUInMemoryLongRun";
+        String type = "awesomeTypeInMemory";
+        String loggerName = "InMemoryLongRun";
+        int drainTimeout = 1;
+        final int msgCount = 100000;
+        HttpsRequestConfiguration conf = getHttpsRequestConfiguration(token, type, mockListener.getPort());
+        SenderStatusReporter reporter = new LogzioTestStatusReporter(logger);
+        LogzioSender logzioSender =
```

Comment:
You have to cut down the repeatable code here to enable the reader to read the actual test scenario.